Process Process-1:
Traceback (most recent call last):
  File "C:\Users\jorge\AppData\Local\Programs\Python\Python38\lib\multiprocessing\process.py", line 315, in _bootstrap
    self.run()
  File "C:\Users\jorge\AppData\Local\Programs\Python\Python38\lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "D:\Code\Python\AnomalyDetection-PyTorch\src\main.py", line 78, in call_training
    myNet.train(resultPath,wandbObject)
  File "D:\Code\Python\AnomalyDetection-PyTorch\src\models\RegularizedEmbeddingClass.py", line 56, in train
    trainLoss, trainLossMSE, trainPSNR, trainSSIM = 0
TypeError: cannot unpack non-iterable int object
[31mStart to train the network